test/units/galaxy/test_collection.py

```diff
@@ -467,7 +467,11 @@ def test_build_existing_output_without_force(collection_input):
         collection.build_collection(to_text(input_dir, errors='surrogate_or_strict'), to_text(output_dir, errors='surrogate_or_strict'), False)
 
 
-def test_build_existing_output_with_force(collection_input):
+@pytest.mark.parametrize("path_sufix", [
+    'test-ÅÑŚÌβŁÈ Collections Input 1 with_slash/',
+    'test-ÅÑŚÌβŁÈ Collections Input 2 no slash',
+])
```

Comment:
This will indeed make sure we test the build collection build with both a normalized path ending in '/' and without.

Reply:
Marking this thread as resolved.